tests/python_tests/test_kv_cache_eviction.py

```diff
@@ -265,3 +267,98 @@ def test_optimized_generation_longbench(device, test_struct):
     assert ref_score - score <= test_struct.threshold
     assert max_optimization_ratio >= test_struct.max_cache_usage_optimization_ratio
     assert avg_optimization_ratio >= test_struct.avg_cache_usage_optimization_ratio
+
+
+MILEBENCH_CACHE_EVICTION_CONFIG = CacheEvictionConfig(start_size=32, recent_size=64, max_cache_size=352, aggregation_mode=AggregationMode.SUM)
+
+@pytest.mark.nightly
+@pytest.mark.parametrize("device", ["CPU", "GPU"])
+@pytest.mark.parametrize(
+    ("test_struct", "download_test_content"), [
+    (BenchmarkTestData("ALFRED", 0.011, 1.440, 1.574), "MileBench_part0.tar.gz"),
+    (BenchmarkTestData("MMCoQA", 0.032, 1.843, 1.620), "MileBench_part2.tar.gz"),
+    (BenchmarkTestData("WikiVQA", 0.032, 1.412, 1.527), "MileBench_part5.tar.gz"),
+    ],
+    indirect=["download_test_content"]
+)
+def test_optimized_generation_milebench(device, test_struct, download_test_content):
```

Comment:
Should it be a part of WWB instead?

Reply:
Here's another PR about multiple-choice: https://github.com/openvinotoolkit/openvino.genai/pull/2254

Reply:
Do you have an example of how to measure a model's actual accuracy metrics independently, without comparing it to the original model?

Reply:
No

Reply:
@Wovchena, @l-bat WWB already supports VLM and does not require MileBench for VLMs validation. As far as I understand, this pr only introduces the regression test for the eviction based on milebench. There are no plans to use this in the regular validation like wwb. 

I do not support the idea of ​​adding metric calculations on the dataset in wwb, but if you are not satisfied with the fact that the code that calculates the metrics on milebench is in the tests, then you can move it to tool as a separate script and use it in the tests.

@Wovchena let's discuss it offline if you have other ideas.